src/components/UserInfo.js

```diff
@@ -12,7 +13,12 @@ export default class UserInfo {
   }
 
   setUserInfo(data) {
-    this._profileName.textContent = data.title;
+    this._profileName.textContent = data.name;
     this._description.textContent = data.description;
+    this._avatar.src = data.avatar;
```

Comment:
`    this._avatar.src = data.avatar;
`

Please, use method `setAvatar` here not to duplicate the same code

Reply:
Fixed.